msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -536,6 +536,9 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 	</Target>
 
 	<Target Name="_DetectSdkLocations" DependsOnTargets="_ComputeTargetArchitectures">
+		<Error Text="Support for classic applications (using monotouch.dll) is obsoleted in Xamarin.iOS 10.0+"
```

Comment:
`Classic` should be capitalized. Also I think it'll be better to tell the user the solution as well, and not only the problem.

So something like this:

    Support for Classic projects is obsolete in Xamarin.iOS 10+. Please migrate your project to Unified.